__tests__/app.test.js

```diff
@@ -141,9 +141,51 @@ describe('App', () => {
       .then((response) => {
         const body = response.body;
         expect(body.articles).toBeSortedBy('created_at', {
-          descending: true,
+          descending: true
         });
       })
     });
   });
+
+  describe('Get /api/articles/:article_id/comments', () => {
```

Comment:
Do you have error handling tests for your 400 and 404?

Reply:
Slipped my mind to add these, have since added them in